crates/wash-cli/src/app/mod.rs

```diff
@@ -67,10 +67,14 @@ pub struct ListCommand {
 pub struct UndeployCommand {
     /// Name of the application to undeploy
     #[clap(name = "name")]
-    app_name: String,
+    app_name: Option<String>,
```

Comment:
Would be good to include a clap https://docs.rs/clap/latest/clap/struct.Arg.html#method.required_if_eq to make sure it's supplied if `--all` isn't

Reply:
Thanks this is a great catch -- I went with the `required_unless_present`!